app/models/outgoing_message.rb

```diff
@@ -470,6 +470,11 @@ def template_changed
     end
   end
 
+  def regex_escape(content)
+    text = content.gsub("\r", "\n") #shouldn't happen, but just in case
```

Comment:
I _think_ you'd want `"\r\n"` as the first argument here. It doesn't technically matter as you're squashing them down later anyway, but it would make the intent clearer.